bundle/package.json

```diff
@@ -99,4 +99,4 @@
 		"typescript": "^4.1.2",
 		"ws": "^8.8.1"
 	}
-}
```

Comment:
:eyes:

Reply:
I can remove that, it did it when i ran `npm run setup` kek